src/main/java/bisq/price/spot/ExchangeRateService.java

```diff
@@ -237,7 +237,7 @@ private Map<String, Object> getMetadata(ExchangeRateProvider provider) {
 
     private long getTimestamp(ExchangeRateProvider provider, Set<ExchangeRate> exchangeRates) {
         return exchangeRates.stream()
-                .filter(e -> provider.getName().equals(e.getProvider()))
+                .filter(e -> provider.getName().startsWith(e.getProvider()))
```

Comment:
@alvasw, needs to be [this way around](https://github.com/bisq-network/bisq-pricenode/pull/30#discussion_r1327763329):

     .filter(e -> e.getProvider().startsWith(provider.getName()))

Otherwise it would log an error and we'd not get the timestamp in metadata.